tutor/src/screens/teacher-gradebook/table.js

```diff
@@ -438,20 +446,22 @@ const Aggregates = observer(({ ux }) => {
         </Cell>
         {ux.headings.map((h, i) => (<MinMaxResult key={i} data={h} ux={ux} type={MinMaxType.MIN} />))}
       </Row>
-    </>
+    </StyledAggregateData>
   );
 });
 
 const GradebookTable = observer(({ ux }) => {
+  // calculating the size of the table
+  const size = useWindowSize();
```

Comment:
Hmm I wonder if we could use CSS's `vh` instead of JS here. I might try that in my branch.